### PR TITLE
Add documentation for simulation API

### PR DESCRIPTION
**Pull request by shahnami**
Adds documentation for the simulation API.

### CODE REVIEW

File: docs/modules/ROOT/pages/admin-api-reference.adoc [hunk 1]
```diff
@@ -179,6 +179,53 @@ The same call can be made using `defender-admin-client` passing the contract and
...
+const data = contractInterface.encodeFunctionData(proposal.functionInterface.name, proposal.functionInputs);
+
+const simulation = await client.simulateProposal(
+  proposal.contractId, // contractId
+  proposal.proposalId, // proposalId
+  {
+    transactionData: {
+      // this is the default hardhat address
+      from: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266', // change this to impersonate the `from` address
+      type: 'function-call', // or 'send-funds'
```
Comment (ernestognw) [inline docs/modules/ROOT/pages/admin-api-reference.adoc:203]: Is this actually used? I looked at the code and I don't see it's relevant anywhere.
We can keep it, it just drew my attention

**Review by ernestognw — APPROVED** [review]

**Review by pasevin — APPROVED** [review]

File: docs/modules/ROOT/pages/admin-api-reference.adoc (hunk 1 above)
Comment (shahnami) [reply docs/modules/ROOT/pages/admin-api-reference.adoc:203]: Good catch! Indeed, doesn't look like we use it anywhere..

**Review by pasevin — APPROVED** [review]

File: docs/modules/ROOT/pages/admin-api-reference.adoc [hunk 2]
```diff
@@ -179,6 +179,54 @@ The same call can be made using `defender-admin-client` passing the contract and
...
+[[simulate-endpoint]]
+=== Simulate a Proposal
+
+Proposals can be simulated via the API. The results of a simulation (`SimulationResponse`) will be stored and can be retrieved with the `getProposalSimulation` endpoint.
+
+```js
+const proposal = await client.getProposal(contractId, proposalId);
+
+// import the ABI and create an ethers interface
+const contractInterface = new utils.Interface(contractABI);
```
Comment (mok0230) [inline docs/modules/ROOT/pages/admin-api-reference.adoc:191]: Thoughts on

```suggestion
const contractInterface = new ethers.utils.Interface(contractABI);
```

?

I know it says ethers in the comment directly above but when I first saw this I thought it was some utils associated with the `defender-client` package

**Review by mok0230 — APPROVED** [review]
looks great!

**Review by mok0230 — APPROVED** [review]

**Pull request merged**